Sasquatch/SasquatchObjC/AppDelegate.h

```diff
@@ -6,5 +6,6 @@
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 @property(strong, nonatomic) UIWindow *window;
+- (void)requestLocation;
```

Comment:
Empty line between lines.
```suggestion

- (void)requestLocation;
```